src/components/CardAboutShelter/CardAboutShelter.tsx

```diff
@@ -106,16 +108,22 @@ const CardAboutShelter = (props: ICardAboutShelter) => {
           value={check(shelter.pix) ? `${shelter.pix}` : 'Não informado'}
           clipboardButton={check(shelter.pix)}
         />
-      {(petsRsShelterUrl != '') ? (
+        {petsRsShelterUrl != '' ? (
```

Comment:
Aqui poderia ser assim não?
```suggestion
        {petsRsShelterUrl != '' && (
```